mne/preprocessing/maxwell.py

```diff
@@ -678,6 +650,207 @@ def _sss_basis(origin, coils, int_order, ext_order, mag_scale=100.,
     return S_tot
 
 
+def _sss_basis(origin, coils, int_order, ext_order, mag_scale=100.):
+    """Compute SSS basis for given conditions.
+
+    Parameters
+    ----------
+    origin : ndarray, shape (3,)
+        Origin of the multipolar moment space in millimeters
+    coils : list
+        List of MEG coils. Each should contain coil information dict specifying
+        position, normals, weights, number of integration points and channel
+        type. All coil geometry must be in the same coordinate frame
+        as ``origin`` (``head`` or ``meg``).
+    int_order : int
+        Order of the internal multipolar moment space
+    ext_order : int
+        Order of the external multipolar moment space
+    mag_scale : float
+        Scale factor for magnetometers.
+
+    Returns
+    -------
+    bases : ndarray, shape (n_coils, n_mult_moments)
+        Internal and external basis sets as a single ndarray.
+
+    Notes
+    -----
+    Incorporates magnetometer scaling factor. Does not normalize spaces.
+
+    Adapted from code provided by Jukka Nenonen.
+    """
+    # Get position, normal, weights, and number of integration pts.
+    rmags, cosmags, wcoils, bins = _concatenate_coils(coils)
+    rmags -= origin
+    # Convert points to spherical coordinates
+    rad, az, pol = _cart_to_sph(rmags).T
+    cosmags *= wcoils[:, np.newaxis]
+    del wcoils
+    out_type = np.float64
+    n_coils = len(coils)
+
+    # Set up output matrices
+    n_in, n_out = _get_n_moments([int_order, ext_order])
+    S_tot = np.empty((len(coils), n_in + n_out), out_type)
+    S_in = S_tot[:, :n_in]
+    S_out = S_tot[:, n_in:]
+
+    # Set all magnetometers (with 'coil_class' == 1.0) to be scaled by 100
```

Comment:
To fit it all in one line:
Scale all magnetometers (with `coil_class` == 1.0) by `mag_scale`